apps/src/p5lab/AnimationPicker/AnimationPickerListItem.jsx

```diff
@@ -90,11 +91,18 @@ class AnimationPickerListItem extends React.Component {
       }
     ];
 
-    const iconImageSrc = this.props.category
-      ? `/blockly/media/gamelab/animation-previews/category_${
-          this.props.category
-        }.png`
-      : '';
+    // TODO(JillianK): Remove this additional isSpriteLab logic once new sprites have been added to the SpriteLab animation library.
```

Comment:
you would only have one TODO if you moved this calculation into `<P5LabView/>` (i.e., instead of passing `isSpritelab` as the prop, you'd pass something like `categoryImagePathPrefix`). this is all temporary, so i'm not sure it matters much